communication/vehicleconnections/mavsdkstation.h

```diff
@@ -30,12 +33,19 @@ class MavsdkStation : public QObject
 
     QList<QSharedPointer<MavsdkVehicleConnection>> getVehicleConnectionList() const;
 
+private slots:
+    void on_gotHeartbeat(quint8 systemId);
+    void on_timeout();
+
 signals:
     void gotNewVehicleConnection(QSharedPointer<MavsdkVehicleConnection> vehicleConnection);
```

Comment:
```suggestion
    void gotNewVehicleConnection(QSharedPointer<MavsdkVehicleConnection> vehicleConnection);
    void disconnectOfVehicleConnection(int vehicleID);
```

Reply:
I just realized during testing that the disconnected vehicle is not removed from the map, because the map is not notified of the disconnect. I think a simple solution would be to add a signal which contains the vehicleID. Then we need to connect to this signal in ControlTower's mainwindow.